chroma_core/lib/util.py

```diff
@@ -263,7 +313,9 @@ def start_action(ActionResult, trigger):
     # check cancel_event
     while True:
         if cancel_event.is_set():
-            requests_unixsocket.post(SOCKET_PATH, json=(host, {"type": "ACTION_CANCEL", "id": str(request_id)})).content
+            requests_unixsocket.post(
```

Comment:
This will need to be updated for docker.

Reply:
Yes, but your patch hasn't landed yet.

Whichever patch lands first, the other will need to rebase and update..